### PR TITLE
create thala-sdk monorepo

### PR DESCRIPTION
**Pull request by SamuelQZQ**
- build dependency handled automatically. for example, router depends on math, it will always build math first, then router.
- Versioning handled automatically. for example, router depends on math, whenever math release a new version, router will also release a new version which using the new math.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# thala-sdk
+
+- [@thalalabs/thalaswap-math](./packages/thalaswap-math/)
+- [@thalalabs/router-sdk](./packages/thalaswap-router/)
```
Comment (0xbe1) [inline Readme.md:4]: can we name router-sdk to thalaswap-router? not sure how easy is it to rename a npm package

**Review by 0xbe1 — APPROVED** [review]
lgtm

File: Readme.md (hunk 1 above)
Comment (SamuelQZQ) [reply Readme.md:4]: @0xbe1 it will simply release a new package

Comment (SamuelQZQ) [reply Readme.md:4]: anyway, merge it now. We can use new name to publish the package if we want.

**Pull request merged**